helm/examples/azure/hpcc-azurefile/templates/NOTES.txt

```diff
@@ -1,8 +1,9 @@
 {{ $dllPvcName := printf "dll-%s-pvc" (include "hpcc-azurefile.fullname" $) }}
 {{ $daliPvcName := printf "dali-%s-pvc" (include "hpcc-azurefile.fullname" $) }}
 {{ $dataPvcName := printf "data-%s-pvc" (include "hpcc-azurefile.fullname" $) }}
+{{ $sashaPvcName := printf "sasha-%s-pvc" (include "hpcc-azurefile.fullname" $) }}
 
-# Persistent Volume Claims created: {{ $dllPvcName }}, {{ $daliPvcName }}, {{ $dataPvcName }}
+# Persistent Volume Claims created: {{ $dllPvcName }}, {{ $daliPvcName }}, {{ $dataPvcName }}, {{ $sashaPvcName }}
```

Comment:
Needs to adhere to what planes listed in this charts values.yaml.
See examples/local/hpcc-localfile/NOTES.txt

Reply:
The output from NOTES.txt (i.e. the output from helm install azstorage ./examples/azure/hpcc-azurefile) minus the header lines, should exactly match the values-retained-azurefile.yaml